content/blog/2018/06/2018-05-14-simple-pull-request-plugin.adoc

```diff
@@ -0,0 +1,271 @@
+---
+layout: post
+title: "GSoC Project Intro: Pipeline as YAML"
+tags:
+- gsoc2018
+- plugin
+- pipeline
+- yaml
+author: abhishek_gautam
+---
+
+=== About me
+
+I am Abhishek Gautam, 3rd year student from Visvesvaraya National Institute of
+technology, India, Nagpur. I was a member of ACM Chapter and Google student developer club of my
+college. I am passionate about automation.
+
+=== Mentors
+
+* link:https://github.com/oleg-nenashev[Oleg Nenashev] (Org Admin)
+* link:https://github.com/martinda[Martin d'Anjou]
+* link:https://github.com/kwhetstone[Kristin Whetstone]
+* link:https://github.com/grandvizier[Jeff Knurek]
+
+=== Project Summary
+
+This is a GSoC 2018 project.
+
+This project aims to develop a Pull request Job Plugin. Users should be able to
```

Comment:
> develop a Pull request Job Plugin

sometimes "pull request" is with lower case letters and sometimes it's with upper case "P". If using uppercase "P" than the "R" should also be uppercase. You should check the rest of this doc to make it consistent. 

However in this usage, I'm not sure if any of those letters should be uppercase, as you're describing what the project is creating, not specifically the name of the project.